.stylelintrc

```diff
@@ -12,6 +12,10 @@
       true,
       { "parser": "css" }
     ],
-    "length-zero-no-unit": null
+    "alpha-value-notation": null,
+    "function-url-quotes": null,
+    "keyframes-name-pattern": null,
+    "length-zero-no-unit": null,
+    "selector-class-pattern": null,
```

Comment:
A few of the new rules are not really adapted to our codebase, or we don't really care (my opinion :-) ).

`alpha-value-notation` wants all alpha values to use percentages (or number if we configure it this way). That is opacity, and in rgb() values, and everywhere else too. I don't really care so I disabled it.
`function-url-quotes` requires quotes in `url("path/to/image.jpg")` but we don't do that most of the time.
`keyframes-name-pattern` and `selector-class-pattern` want that we use kebab case of all classes and keyframes name. While I think this is a good idea, we haven't done this really, and it would be painful to change now.
`length-zero-no-unit` was there before. Note that `length-zero-no-unit` conflicts a little bit with the new rule `hue-degree-notation` that requests that hue uses a unit always, even with 0... I wasn't sure about that new one...